vispy/visuals/collections/agg_fast_path_collection.py

```diff
@@ -213,6 +213,7 @@ def bake(self, P, key='curr', closed=False, itemsize=None):
 
     def draw(self, mode="triangle_strip"):
         """Draw collection"""
-        gl.glDepthMask(gl.GL_FALSE)
+        # Would call context.set_depth_mask() here, but we have no access
```

Comment:
Do we not have access to it because this isn't a "real" Visual with access to those `set_gl_state`-like methods?

Reply:
Don't we have access (or can't we get it) by using the gloo functions that is in the `wrappers.py` you updated in this PR?

Reply:
I thought we could not access it, but indeed we can  just call `gloo.set_depth_mask()` :)